app/src/androidTest/java/com/github/se/signify/ui/screens/UtilsTest.kt

```diff
@@ -101,4 +104,113 @@ class UtilsTest {
     // Assert the click listener was triggered
     assert(clickCounter == 1)
   }
+
+  @Test
+  fun squareButton_isDisplayed_andClickable() {
+    val label = "Test Button"
+    composeTestRule.setContent {
+      SquareButton(
+          iconRes = R.drawable.battleicon,
+          label = label,
+          onClick = {},
+          size = 100.dp,
+          iconSize = 50.dp,
+          labelFontSize = 16.sp,
+          iconTint = Color.Gray,
+          textColor = Color.White,
+          modifier = Modifier.testTag(label))
+    }
```

Comment:
As said in the previous PR review, when you make test and you have to set a function with basic value and you will reuse it, you can make a setting function and just call the setting function in the specific test to avoid duplication.